content/blog/2022-12-06-keda-2.9.0-release.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Announcing KEDA v2.9 🎉"
+date = 2022-12-08
+author = "Jeff Hollan (KEDA), Tom Kerkhove (KEDA)"
+aliases = [
+"/blog/2.9.0-release"
+]
++++
+
+We recently completed our most recent release: 2.9.0 🎉!
+
+Here are some highlights:
```

Comment:
Also, we are enforcing our Kubernetes compatability policy which means end-users must use Kubernetes v1.23 at least

https://keda.sh/docs/2.9/operate/cluster/#kubernetes-compatibility